spec/support/helpers/cli_helper.cr

```diff
@@ -40,7 +40,7 @@ module CLIHelper
   def expected_db_url(db_key, env)
     case db_key
     when "pg"
-      "postgres://#{`whoami`.strip("\n")}:@localhost:5432/#{TEST_APP_NAME}_#{env}"
```

Comment:
I just added the `whoami` interpolation last week because it makes generating a new app and getting started pretty fast. Does this effect the tests on docker and that's why you added the `postgres:password` back? I had wondered, but I don't use docker and I forgot to check (I'm not a docker wizard).

Reply:
yeah, it breaks the docker but also my local setup.  :-)